src/lib/grid/row.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, Input, HostBinding, ViewEncapsulation, Directive, ChangeDetectionStrategy } from '@angular/core';
+
+@Directive({
+    selector: 'apto-col'
+})
+export class AptoGridColumnDirective {
+    @Input() public xs: number|string|boolean;
```

Comment:
Why the amount of variance?

Reply:
so you can set the value to [1-9] 'auto' or true based on what you want to do with your grid